File: JavaScript/Gate Two/TaskNine.js
const prompt = require('prompt-sync')();

let height = Number(prompt("Enter height of triangle: "));

for (let count = 1; count <= height; count++){

	let star = "";

	for (let counter = 1; counter <= count; counter++){
		star += "*"
	}
	console.log(star);

}

File: Java/Gate Two/TaskNine.java
import java.util.Scanner;
public class TaskNine{

	public static void main (String [] args){
	
		Scanner input = new Scanner (System.in);

		System.out.println("Enter triangle height: ");
		int height = input.nextInt();

		for (int count = 1; count <= height; count++){
			System.out.println("*".repeat(count));
		}
	}

}